### PR TITLE
chore: Add attempt count changes

### PR DESCRIPTION
**Pull request by thisisabhash**
*Issue #, if available:*

*Description of changes:*
Add attempt count changes and update UI for result view in HostApp
 
*Check points: (check or cross out if not relevant)*

- [ ] Added new tests to cover change, if needed
- [x] Build succeeds with all target using Swift Package Manager
- [x] All unit tests pass
- [x] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [x] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ let package = Package(
     ],
     dependencies: [
         // TODO: Change this before merge to main
-        .package(url: "https://github.com/aws-amplify/amplify-swift", branch: "feat/no-light-support")
+        .package(url: "https://github.com/aws-amplify/amplify-swift", branch: "chore/attempt-count-changes")
```
Comment (phantumcode) [inline Package.swift:17]: does this still need to be updated?

File: Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel.swift [hunk 2]
```diff
@@ -186,13 +193,20 @@ class FaceLivenessDetectionViewModel: ObservableObject {
 
     func initializeLivenessStream() {
         do {
-            guard let livenessSession = livenessService as? FaceLivenessSession else {
-                throw FaceLivenessDetectionError.unknown
+            if (abs(Self.attemptIdTimeStamp.timeIntervalSinceNow) > defaultAttemptCountResetInterval) {
+                Self.attemptCount = 1
```
Comment (phantumcode) [inline Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel.swift:197]: should this be initially 0 or 1?

File: Sources/FaceLiveness/Views/GetReadyPage/GetReadyPageView.swift [hunk 3]
```diff
@@ -27,28 +27,19 @@ struct GetReadyPageView: View {
...
-                    }.padding()
-                }
+                VStack {
+                    WarningBox(
+                        titleText: LocalizedStrings.get_ready_photosensitivity_title,
+                        bodyText: LocalizedStrings.get_ready_photosensitivity_description,
+                        popoverContent: { photosensitivityWarningPopoverContent }
+                    )
+                    .accessibilityElement(children: .combine)
+                    .opacity(challenge.type == .faceMovementAndLightChallenge ? 1.0 : 0.0)
```
[the remark this answers is not in the document]
Comment (thisisabhash) [reply Sources/FaceLiveness/Views/GetReadyPage/GetReadyPageView.swift:37]: I used the concept from here about conditionally hiding the view (but still taking the space) : https://developer.apple.com/tutorials/swiftui-concepts/choosing-the-right-way-to-hide-a-view

Idea is for `Center your face text` to be placed right above the camera oval (and not be at top of the screen)

File: Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel.swift (hunk 2 above)
Comment (thisisabhash) [reply Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel.swift:197]: it should be 1 as whenever `initializeLivenessStream()` is called - either that attempt is the first attempt for new interval or adds to previous attempt count

**Review by phantumcode — APPROVED** [review]

**Pull request merged**